Puppetfile.lock

```diff
@@ -1,12 +1,12 @@
 FORGE
   remote: https://forgeapi.puppetlabs.com
   specs:
-    camptocamp-systemd (1.0.0)
+    camptocamp-systemd (1.1.1)
```

Comment:
Unless there's an issue I know that should be fixed I usually keep it at the older version for third part releases, just to be safe.